sirbot/core/registry.py

```diff
@@ -53,3 +53,5 @@ def __len__(self):
 
     def __contains__(self, item):
         return item in self._plugins
+
+Registry = RegistrySingleton()
```

Comment:
haha I named it just `registry` at first and then decided to change to see how it would look. I guess it looks bad :D